tests/utils/shippable/shippable.sh

```diff
@@ -118,8 +118,10 @@ function cleanup
             ansible-test coverage xml --color --requirements --group-by command --group-by version ${stub:+"$stub"}
             cp -a tests/output/reports/coverage=*.xml "$SHIPPABLE_RESULT_DIR/codecoverage/"
 
-            # analyze and capture code coverage aggregated by integration test target
-            ansible-test coverage analyze targets generate -v "$SHIPPABLE_RESULT_DIR/testresults/coverage-analyze-targets.json"
+            # analyze and capture code coverage aggregated by integration test target if not on 2.9
+            if [ -n "${ANSIBLE_BASE_REV:-}" ] || [ "${ANSIBLE_BASE_REV:-}" != "2.9" ]; then
```

Comment:
It should just be:
`if [ "${ANSIBLE_BASE_REV:-}" != "2.9" ]; then`

Otherwise, `ANSIBLE_BASE_REV` is not empty, and consequently the left block is true.

Reply:
The default is to be empty, today it might be set to 2.9 but in the very near future it could also be 2.10.  So the goal is 'variable is empty OR is not 2.9'

Reply:
You want to run `ansible-test coverage analyze targets generate` if `ANSIBLE_BASE_REF` is empty or if it's value is `2.9`. Right?

Reply:
No, I want to run it if the value is empty or not 2.9 - the command wasn't added until 2.10.

Reply:
(Empty ends up being devel)

Reply:
Oh! thanks for the clarification :-)

Reply:
I'll update the comment to make it more clear before I merge, thanks for reviewing @goneri!